src/test/java/ttt/PromptSpy.java

```diff
@@ -0,0 +1,44 @@
+package ttt;
+
+public class PromptSpy implements Prompt {
+    private Board lastBoardPrinted;
+    private String lastMessagePrinted;
+
+    @Override
+    public String read() {
+        return null;
+    }
+
+    @Override
+    public void askUserForTheirMove() {
+
+    }
+
+    @Override
+    public void print(Board board) {
+        this.lastBoardPrinted = board;
+    }
+
+    @Override
+    public void printWinningMessage() {
+        lastMessagePrinted = "Congratulatory message printed";
```

Comment:
At the very least, these "test" messages should be made into constants so that the tests aren't coupled to the details in here.